hazelcast/src/main/java/com/hazelcast/multimap/impl/MultiMapProxyImpl.java

```diff
@@ -390,6 +390,9 @@ public LocalMultiMapStats getLocalMultiMapStats() {
         for (Data dataKey : dataSet) {
             keySet.add((K) nodeEngine.toObject(dataKey));
         }
+        if (config.isStatisticsEnabled()) {
+            getService().getLocalMultiMapStatsImpl(name).incrementOtherOperations();
+        }
```

Comment:
Why should we have this here and not in the operation-related tasks?

Reply:
the stat updates of other operation-related tasks are hidden in [MultimapProxySupport](https://github.com/hazelcast/hazelcast/pull/19296/files/b1e67445d2a21bf033acf87c7c35ff3f9f5eaa44#diff-3bc1f02702443408f8f695b5d473ee987730fd73bd82de575088db031d257c85R447-R459) class.
I couldn't do this in a consistent way with the others because the call stack is a little different from the others